assets/src/application/ui/layout/entityList/filterBar/buttons/FilterButtonWrap.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
+import { withViewportMatch } from '@appDisplay/viewport';
 import { __ } from '@wordpress/i18n';
 
 import { FilterButtonWrapProps } from '../types';
 
-const FilterButtonWrap: React.FC<FilterButtonWrapProps> = ({ children, id, label }) => {
-	return (
-		<div className='ee-filter-bar__btn-wrap'>
-			{children}
+const FilterButtonWrap: React.FC<FilterButtonWrapProps> = ({ children, id, isMobile, label }) => (
+	<div className='ee-filter-bar__btn-wrap'>
+		{children}
+		{!isMobile && (
 			<label className='ee-filter-btn-label' htmlFor={id}>
 				{label}
 			</label>
```

Comment:
why are you removing this label on mobile?
isn't this for adding the labels like "table view", "card view", etc ?

Reply:
without that we'll have this doubling label situation;

![image](https://user-images.githubusercontent.com/1477453/77585969-0d24aa80-6eee-11ea-8514-92c2508ba1b0.png)

![image](https://user-images.githubusercontent.com/1477453/77586028-2299d480-6eee-11ea-9390-911a6bcb5bc4.png)

Reply:
ahhh ok... ya I get it now, didn't realize the extra label was being added to those too